packages/init/__tests__/utils/create-repo.test.ts

```diff
@@ -4,25 +4,27 @@ import nock from "nock";
 import { createRepository } from "../../src/utils/create-repo";
 import { stderr, stdout } from "stdout-stderr";
 import { Models } from "@slicemachine/core";
+import { Client } from "../../src/utils";
+import { ApplicationMode } from "@slicemachine/client";
+
+Client.initialize(ApplicationMode.PROD, "theBatman");
```

Comment:
We don't need it.
Each file seems to be test independently, if you try removing this line in one of the test suite, you'll see that the tests are not passing anymore.
In Slice Machine and in the Init, we do not need to destroy it.
In the tests, we also do not need to destroy it because we use the same authentication token / applicationMode